db/migrate/20240826165003_add_delete_cascade_for_sources_and_initiatives.rb

```diff
@@ -0,0 +1,5 @@
+class AddDeleteCascadeForSourcesAndInitiatives < ActiveRecord::Migration[7.1]
+  def change
+    add_foreign_key :initiatives, :wyeworkers, column: :owner_id, on_delete: :cascade
```

Comment:
This is creating a duplicated constraint you need to delete the original

Reply:
Thank you! Fixed!